docs/specifications/tests/DNSSEC-TP/dnssec11.md

```diff
@@ -1,50 +1,244 @@
+  (only if *Test Type* is [undelegated]).
+
+
+## Summary
+
+* 
+
+Message Tag outputted         | Level   | Arguments  | Description of when message tag is outputted
+:-----------------------------|:--------|:-----------|:--------------------------------------------
+DS11_INCONSISTENT_DS          | WARNING |            | Parent name servers are inconsistent on the existence of DS.
+DS11_INCONSISTENT_SIGNED_ZONE | ERROR   |            | Name servers for the child zone is inconsitent on wherer the zone is signed or not.
+DS11_INDETERMINED_DS          | ERROR   |            | It cannot be determined if the parent zone as DS for the child zone or not.
+DS11_INDETERMINED_SIGNED_ZONE | ERROR   |            | It cannot be determined if the child zone is signed or not.
+DS11_PARENT_WITHOUT_DS        | NOTICE  | ns_ip_list | List of parent name servers without DS for the child zone.
+DS11_PARENT_WITH_DS           | NOTICE  | ns_ip_list | List of parent name servers with DS for the child zone.
+DS11_NS_WITH_SIGNED_ZONE      | NOTICE  | ns_ip_list | List of child name servers with signed child zone.
+DS11_NS_WITH_UNSIGNED_ZONE    | WARNING | ns_ip_list | List of child name servers with unsigned child zone.
+DS11_DS_BUT_UNSIGNED_ZONE     | ERROR   |            | The child zone is unsigned, but the parent zone has DS record.
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+
+## Test procedure
+
+1.  Create the following empty sets:
+    1. Parent name server IP address ("Indetermined DS").
+    2. Parent name server IP address ("No DS Record").
+    3. Parent name server IP address ("Has DS Record").
+    4. Child name server IP address ("Indeterminded DNSKEY")
+    5. Child name server IP address ("No DNSKEY Record").
+    6. Child name server IP address ("Has DNSKEY Record").
+
+2.  If *Test Type* is "normal", then:
+
+    1. Create a DS query with the DO flag set for the name of the *Child Zone*
+       ("DS Query").
+
+    2. Retrieve all name server IP addresses for the parent zone of
+       *Child Zone* using [Method1] ("Parent NS IP").
+
+    3. For each name server IP in *Parent NS IP* do:
+
+       1. Send *DS Query* to the name server IP.
+       1. If the response has the TC flag set, re-query over TCP and use that
+          response instead.
+       2. If there is no DNS response, then add the name server (IP) to the
+          *Indetermined DS* set.
+       3. Else, if the RCODE of response is not "NoError" ([IANA RCODE List]),
+          then add the name server (IP) to the *Indetermined DS* set.
+       4. Else, if the AA flag is not set in the response, then add the name
+          server (IP) to the *Indetermined DS* set.
```

Comment:
I liked how you grouped these 3 points in other PRs. Would you mind doing the same here?

Reply:
It seems that the same action is performed here: "add the name server to the *Undetermined DS* set". Can't this action be distributed? Ofc both are valid. This is just a remark on the style.

And for the other PRs where you regrouped these points together, we can say that this is because you perform the same action: "skip this name server and go to the next one".

Reply:
OK, I will group.